moon/config/moon-excavator-round2.json

```diff
@@ -75,6 +84,9 @@
               ["rosmsg.vslam_enabled", "app.vslam_enabled"],
               ["rosmsg./qual_2_score", "app.score"],
               ["rosmsg.joint_name", "excavator.joint_name"],
+              ["rosmsg.left_image", "detector.left_image"],
+              ["rosmsg.left_image", "app.left_image"],
```

Comment:
why application gets only left image? Is it intent?

Reply:
it's used for detecting the other robot and for dynamic lighting adjustments, no stereo needed for either